camptocamp/ngeo
# No more use OpenLayers eslint config

- Remove dependency on `eslint-config-openlayers`
- Test all the js files
- Set some warning on js files as error on ts files (is the IDE you will see them as warning)
- Apply all the fix
